hail/python/hailtop/batch/backend.py

```diff
@@ -208,36 +208,48 @@ class ServiceBackend(Backend):
     Examples
     --------
 
-    >>> service_backend = ServiceBackend('test')
-    >>> b = Batch(backend=service_backend)
+    >>> service_backend = ServiceBackend('my-billing-account', 'my-bucket') # doctest: +SKIP
+    >>> b = Batch(backend=service_backend) # doctest: +SKIP
     >>> b.run() # doctest: +SKIP
-    >>> service_backend.close()
+    >>> service_backend.close() # doctest: +SKIP
 
-    If the Hail configuration parameter batch/billing_project was previously set
-    with ``hailctl config set``, then one may elide the billing_project
-    parameter.
+    If the Hail configuration parameters batch/billing_project and
+    batch/bucket were previously set with ``hailctl config set``, then
+    one may elide the billing_project and bucket parameters.
 
-    >>> service_backend = ServiceBackend()
-    >>> b = Batch(backend=service_backend)
+    >>> service_backend = ServiceBackend() # doctest: +SKIP
+    >>> b = Batch(backend=service_backend) # doctest: +SKIP
     >>> b.run() # doctest: +SKIP
-    >>> service_backend.close()
+    >>> service_backend.close() # doctest: +SKIP
 
     Parameters
     ----------
     billing_project: :obj:`str`
         Name of billing project to use.
+    bucket: :obj:`str`
+        Name of bucket to use for temporaries.
```

Comment:
It might be good to state explicitly that this should not be in the format gs:// or we should have a check somewhere that asserts the bucket name is valid.